tutor/templates/config.yml

```diff
@@ -62,6 +62,8 @@ OPENEDX_MYSQL_DATABASE: "openedx"
 OPENEDX_CSMH_MYSQL_DATABASE: "{{ OPENEDX_MYSQL_DATABASE }}_csmh"
 OPENEDX_MYSQL_USERNAME: "openedx"
 OPENEDX_COMMON_VERSION: "open-release/lilac.3"
+OPENEDX_EXTRA_PIP_REQUIREMENTS:
```

Comment:
Please document this new setting in configuration.rst. We should mention its existence and give an example on how to use it in the "Installing extra xblocks and requirements" section.

Reply:
Added some documentation for the setting.

Reply:
This setting should also be documented, along with its default value, here: https://docs.tutor.overhang.io/configuration.html#open-edx-customisation